src/library.js

```diff
@@ -1464,6 +1464,8 @@ LibraryManager.library = {
 
   llvm_lifetime_start: function() {},
   llvm_lifetime_end: function() {},
+  llvm_lifetime_start_p0i8: function() {},
+  llvm_lifetime_end_p0i8: function() {},
```

Comment:
I think these should be implemented in `CallHanders.h` in fastcomp, so that we emit no-ops for them (instead of emitting ffi calls that are then no-ops).

Reply:
That is a great idea. Applied that treatment to all of these no-ops.